shanoir-ng-datasets/src/main/java/org/shanoir/ng/dataset/model/DatasetExpression.java

```diff
@@ -57,9 +57,13 @@ public class DatasetExpression extends AbstractEntity {
 	@JoinColumn(name = "dataset_id")
 	private Dataset dataset;
 
+
 	/** Dataset expression format. */
 	private Integer datasetExpressionFormat;
 
+	/** size of the associated dataset files. */
+	private Long size;
```

Comment:
If you add a field, an SQL script to add the column is necessary for the qualif / production server. (but not locally)
Please see what is done in ../shanoir-ng/docker-compose/database/db-changes/datasets

Reply:
All right, i'll add it !